wave_optimization/src/ceres/ceres_examples.cpp

```diff
@@ -0,0 +1,86 @@
+#include "wave/optimization/ceres/ceres_examples.hpp"
+
+namespace wave {
+namespace ceres {
```

Comment:
Seems like a bad idea to have our own namespace called ceres, and then to frequently use the root ceres namespace. Yikes on readability